app/src/test/java/tech/relaycorp/gateway/domain/publicsync/RegisterGatewayTest.kt

```diff
@@ -201,9 +204,10 @@ class RegisterGatewayTest : BaseDataTestCase() {
         )
     }
 
-    private fun buildPNR(publicGatewaySessionKey: SessionKey?) = PrivateNodeRegistration(
+    private fun buildPNR(internetGatewaySessionKey: SessionKey?) = PrivateNodeRegistration(
         PDACertPath.PRIVATE_GW,
-        PDACertPath.PUBLIC_GW,
-        publicGatewaySessionKey
+        PDACertPath.INTERNET_GW,
+        PDACertPath.INTERNET_GW.subjectId,
```

Comment:
This shouldn't be the id, but the Internet address. Since this didn't break any test, can you please see if there's an existing test where it'd be appropriate to add this assertion or, if not, create a new test?